pkg/cmd/admin/registry/registry.go

```diff
@@ -299,6 +302,9 @@ func RunCmdRegistry(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg
 	env := app.Environment{}
 	env.Add(secretEnv)
 
+	if cfg.EnforceQuota {
```

Comment:
If we remove this flag altogether and require it to be enabled with oc
set env for 1.2, we can set it to default to true in a future release
.  I'd prefer not to add flags to the registry for now.